src/fragments/lib/auth/js/manageusers.mdx

```diff
@@ -166,24 +176,61 @@ let result = await Auth.updateUserAttributes(user, {
 });
 console.log(result); // SUCCESS
 ```
+<Callout>
+Note: If you change an attribute that requires confirmation (i.e. email or phone number), you will receive a confirmation code to that attribute.
+</Callout> 
 
-You can delete user attributes:
+`Auth.updateUserAttributes()` function dispatches hub event to help identify attributes that require
+verification. You can listen to `updateUserAttributes` event on `auth` channel:
 
 ```javascript
-let user = await Auth.currentAuthenticatedUser();
+Hub.listen('auth', ({payload}) => {
+    if (payload.event === 'updateUserAttributes') {
+        const resultObject = payload.data;
+    }
+});
+```
+You can learn more about how to listen to Hub events [here](https://docs.amplify.aws/lib/utilities/hub/q/platform/js/).
 
-let result = await Auth.deleteUserAttributes(user, ['family_name']);
-console.log(result); // SUCCESS
+Example of the dispatched result object:
+
+```javascript
+{
+    'email': {
+        isUpdated: false // indicates that attribute requires verification
+        codeDeliveryDetails: {
+            AttributeName: 'email',
+            DeliveryMedium: 'EMAIL',
+            Destination: 'm***@a...'
+        }
+    },
+    'family_name': {
+        isUpdated: true
+    }
+}
 ```
+If user sends several attributes for update and one or more of them is `read_only` or doesn't exist, Amplify JS library will throw
+an error and dispatch `updateUserAttributes_failure` Hub event. None of the attributes in the list will be updated.
 
-> You can find a [list of all custom attributes here](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes).
+### Confirm attribute
 
-If you change the email address, the user will receive a confirmation code. In your app, you can confirm the verification code:
+If you change the email address, you will receive a confirmation code.  In your app, you can confirm your new email address:
```

Comment:
you will receive a confirmation code to the new email address. Below is an example of how to confirm the new email address in your app: